### PR TITLE
makes the mining base placement random

### PR DESCRIPTION
**Pull request by Amelia0010**
### Intent of your Pull Request
image

#### Changelog

:cl:  
tweak: mining base placement is now random
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/mapping.dm [hunk 1]
```diff
@@ -239,7 +239,7 @@ SUBSYSTEM_DEF(mapping)
 
 	// load mining
 	if(config.minetype == "lavaland")
-		LoadGroup(FailedZs, "Lavaland", "map_files/Mining", "Lavaland.dmm", default_traits = ZTRAITS_LAVALAND)
+		LoadGroup(FailedZs, "Lavaland", "yogstation/map_files/mining", "Lavaland.dmm", default_traits = ZTRAITS_LAVALAND)
```
Comment (monster860) [inline code/controllers/subsystem/mapping.dm]: reeeeeeeeee put a //yogs you fudge

**Review by monster860 — DISMISSED** [review]

File: code/controllers/subsystem/mapping.dm (hunk 1 above)
Comment (Amelia0010) [reply code/controllers/subsystem/mapping.dm]: oh yeah, I've beeen doing non-yogs stuff and forgot

Comment (ThatLing) [reply code/controllers/subsystem/mapping.dm]: >:(

**Comment by ThatLing** [comment]
can we nerf mining

**Comment by Nickvr628** [comment]
damn this is pretty cool.

**Comment by Bibby0110** [comment]
description = "The mining base that nanotrasen uses for their mining operations."
nanotrasen. nanotrasen. nanotrasen.

**Comment by Amelia0010** [comment]
🎉

**Comment by HomocidlePancakes** [comment]
Gulag buff. Me no likey. Also gulag teleporter when?

**Comment by Moja10** [comment]
Is the exit of the mining base guaranteed to be rock and not lava floor?

**Comment by Amelia0010** [comment]
yes, and if it weren't there's still the three exits with windows 🗡

**Review by Altoids1 — APPROVED** [review]
The number of things you had to change is suspiciously small.

**Comment by Amelia0010** [comment]
image

**Pull request merged**